.github/workflows/ci.yml

```diff
@@ -42,5 +42,27 @@ jobs:
       - name: Run tests with Node version ${{ matrix.node-version }}
         uses: docker/build-push-action@v2.3.0
         with:
+          tags: calendar-date
           build-args: |
             NODE_VERSION=${{ matrix.node-version }}
+      - uses: shrink/actions-docker-extract@v1
+        id: extract
+        with:
+          image: 'calendar-date'
+          path: '/home/node/coverage/.'
+      - name: Coveralls Parallel
```

Comment:
Please use descriptive names, so one can immediately know what the step is supposed to do when checking results of a pipeline.